cmd/api/main.go

```diff
@@ -157,6 +157,8 @@ func main() {
 			log.Error().Err(err).Msg("closing telemetry module")
 		}
 	})
+
+	log.Print("tableland is ready (temp log, remove later)")
```

Comment:
I could not find this log entry in our [logs](https://console.cloud.google.com/logs/query;query=resource.type%3D%22gce_instance%22%0Aresource.labels.instance_id%3D%22316018482727028278%22;timeRange=2023-01-23T01:35:23.360Z%2F2023-01-23T13:35:23.360Z;cursorTimestamp=2023-01-23T13:35:23.359957254Z?organizationId=733293853667&project=textile-310716). 

Try using the `zerolog` logger, like this:
`log.Info().Msg("tableland is ready (temp log, remove later)")`